docs/learn/documentation/versioned/jobs/configuration-table.html

```diff
@@ -156,13 +156,20 @@
                         others' checkpoints, and perhaps interfere with each other in other ways.
                     </td>
                 </tr>
-                <tr>
                     <td class="property" id="job-coordinator-system">job.coordinator.system</td>
                     <td class="default"></td>
                     <td class="description">
                         <strong>Required:</strong> The <span class="system">system-name</span> to use for creating and maintaining the <a href="../container/coordinator-stream.html">Coordinator Stream</a>.
                     </td>
                 </tr>
+                <tr>
+                    <td class="property" id="job-default-system">job.default.system</td>
```

Comment:
So, I have this longer term dream that most samza users will not have to specify a bunch of .system properties. I see two main use cases:

A) Job creates all streams, IO and samza metadata on the same system. This job could just specify the job.default.system and ignore the others

B) Job creates all IO streams on 1 system, metadata streams on another system. This job would configure
job.metadata.system for all the metadata streams, like we do at LI
job.default.system for all other systems
Override on a case-by-case basis by setting 
streams.x.samza.system

For case A, I think the current naming would make more sense, because it's the default for all streams.

My intent was to update the doc to reflect that when I make that change. But if it's misleading now, I could do it now. Thoughts?